.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -25,61 +25,79 @@ jobs:
     steps:
       - name: "Checkout latest commit"
         uses: actions/checkout@v2
-      - name: "Set up Python 3.8"
+      - name: "Set up Python"
         uses: actions/setup-python@v2
         with:
+          # 3.8 is chosen because it seems to be the fastest for <3.9
+          # (3.9 excluded because it seems to be still very unstable)
           python-version: 3.8
       - name: "Install dependencies"
+        # language=bash
         run: |
-          python -m pip install --upgrade pip
-          pip install tox
-      - name: "Execute tox -e qa,docs"
+          python -m pip install --upgrade pip setuptools wheel
+          pip install colorama pytype flake8 pytest pytest-mock sphinx pickle5 check-manifest
+          python setup.py develop
+      - name: "flake8 Style Checking"
+        # language=bash
         run: |
-          tox -e qa,docs
+          python -m flake8 aiosmtpd conf.py setup.py housekeep.py release.py
+      - name: "Docs Checking"
+        # language=bash
+        run: |
+          pytest aiosmtpd/docs
+          python setup.py build_sphinx
+      - name: "Static Type Checking"
+        run: |
+          pytype --keep-going --jobs auto .
+      - name: "Other QA Checks"
+        # language=bash
+        run: |
+          pytest aiosmtpd/qa
+          check-manifest -v
   testing:
     needs: qa_docs
     strategy:
       # If a matrix fail, do NOT stop other matrix, let them run to completion
       fail-fast: false
       matrix:
-        os: [ "ubuntu-18.04", "ubuntu-20.04", "macos-10.15" ]
-        platform: [ "posix" ]
```

Comment:
Replaced with detection + env var setting in test step proper right before invoking pytest.

Reply:
And it's now moot because we're using `coverage-conditional-plugin`